### PR TITLE
quick fix: change utils to synonym_extractor

### PR DESCRIPTION
**Pull request by Domene99**
Closing #66

### CODE REVIEW

**Comment by Domene99** [comment]
> Hmm, none of the unit tests run across L62? Or there was one but it was failing?

The unit tests run inside synonym_extraction

**Comment by anniefu** [comment]
But the unit tests in here should have broken, right? https://github.com/googleinterns/major-tom/blob/main/services/search/testing/test_search_engine.py

Otherwise, you'll probably want to add a test that would have caught this by testing the SearchEngine constructor.

**Comment by Domene99** [comment]
Turned out to be a quick fix with a big test to back it up

File: services/search/testing/test_search_engine.py [hunk 1]
```diff
@@ -1,11 +1,25 @@
 from unittest import mock
 import json
+import flask  # pylint: disable=import-error
 import responses  # pylint: disable=import-error
 from search_engine import SearchEngine  # pylint: disable=import-error
 import test_constants as constants
+from main import search_service  # pylint: disable=no-name-in-module
```
Comment (ssmall) [inline services/search/testing/test_search_engine.py:7]: if you just did `import main` would you be able to avoid the pylint error?

**Pull request merged**